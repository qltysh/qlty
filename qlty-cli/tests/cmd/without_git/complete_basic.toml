args = [
  "coverage",
  "complete",
  "--dry-run",
  "--override-commit-sha",
  "2ca1bc45a94e37c8dbae6fd9e19fc069ba64bd67",
  "--override-build-id",
  "123",
  "--override-commit-time",
  "2025-05-30T05:00:00+00:00"
]
bin.name = "qlty"
status.code = 0

[env.add]
BUILDKITE_BUILD_NUMBER = "123"
BUILDKITE_BRANCH = "main"
BUILDKITE_BUILD_ID = "build-456"
QLTY_COVERAGE_TOKEN = "123"
QLTY_COVERAGE_TESTING_WITHOUT_GIT = "true"
