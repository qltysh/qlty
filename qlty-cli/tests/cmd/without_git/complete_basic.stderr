qlty [..] [..]-[..] ([..])
[..]
https://qlty.sh/d/coverage

 SETTINGS 

    cwd: [CWD]
    override-commit-sha: 2ca1bc45a94e37c8dbae6fd9e19fc069ba64bd67

 METADATA 

    Commit: 2ca1bc45a94e37c8dbae6fd9e19fc069ba64bd67

 AUTHENTICATION 

    Auth Method: Unknown
    Token: 123

 COMPLETING... 

    Coverage marked as complete in [..]s!

